pages/Configuring/Environmental-variables.md

```diff
@@ -0,0 +1,66 @@
+# Environmental Variables
+
+We have [previously mentioned] using a wrapper to launch Hyprland. Whether you
+start Hyprland through TTY or a Display Manager, it is recommended to use a wrapper.
+As such, you can pass certain environmental variables to improve Wayland compatibility, or
+simply change certain aspects of your desktop.
+
+Please avoid putting those environmental variables in /etc/environment. That will cause all
+sessions (including Xorg ones) to pick up your wayland-specific environmental on traditional
+Linux distros.
+
+## XDG Specifications
+
+- XDG_CURRENT_DESKTOP=Hyprland
```

Comment:
```suggestion
- `XDG_CURRENT_DESKTOP=Hyprland`
```
Same for every other var. It's cleaner to look at.

Reply:
Was actually considered for a while, decided otherwise. Changing now.